docs/sidebars.js

```diff
@@ -33,7 +33,7 @@ const sidebars = {
       },
       items: [
         "articles/step1-installing-cli",
-        "articles/step2-installing-cluster-components",
+        "articles/step2-installing-agent",
```

Comment:
let's not just change the url for now, please, this will have cascading effects.. 
happy to keep it as cluster-components